src/scss/fields.scss

```diff
@@ -1,98 +1,63 @@
-@include oFormsClass(o-forms__field-group) {
+@if not $o-forms-is-silent {
+.o-forms-group {
```

Comment:
For now, not all the properties are available in silent mode.

Considering adding them on a per-class basis as Membership asks for them.

This one is a good candidate though!